docs/ecosystem.md

```diff
@@ -0,0 +1,57 @@
+The following document provides an ecosystem overview of how the various tools compatible with Cirq can work together to enable quantum computing research.
+
+![image alt text](images/ecosystem.png )
+
+* **Research Libraries and Tools:** Libraries and tools used for researching new quantum algorithms and designing and preparing experiments on quantum devices
+
+* **Cirq**: A framework specifically for programming noisy intermediate quantum computers
+
+* **Quantum Cloud Services:** Cirq can connect to a variety of quantum cloud services. Behind each cloud service, one can invoke either an actual quantum computer or a quantum circuit simulator
+
+* **Quantum Circuit Simulators:** Cirq is compatible with a number of quantum circuit simulators that can run either locally or in a distributed fashion
+
+# Research libraries and tools 
+
+## Algorithm libraries and experiments
+
+|Name|Main sponsor|Description|
+|--- |--- |--- |
+|[OpenFermion](https://github.com/quantumlib/OpenFermion)|Google|An open-source algorithms library for developing new quantum chemistry and materials simulation algorithms|
+|[TensorFlow Quantum](https://tensorflow.org/quantum)|Google|An open-source algorithms library for developing new quantum machine learning algorithms|
+|[ReCirq](https://github.com/quantumlib/ReCirq)|Google|An open-source repository of example experiments, tools, and tutorials for running experiments on Google’s Quantum Computing Service|
+|[Forge](https://forge.qcware.com/)|QCWare|A proprietary library for domain-specific quantum algorithms and applications|
+|[Pennylane](https://pennylane.ai/)|Xanadu|An open-source library for quantum machine learning with TensorFlow, PyTorch, or NumPy|
+
+
+
+## Development tools
+
+|Name|Main sponsor|Description|
+|--- |--- |--- |
+|[MITIQ](https://github.com/unitaryfund/mitiq)|Unitary Fund|An open-source library for error mitigation|
+|[Orquestra](https://www.zapatacomputing.com/orquestra/)|Zapata|A proprietary workflow management tool for quantum computing|
+|[pyGSTi](https://www.pygsti.info/)|Sandia National Labs|An open-source library for modeling and characterizing noisy quantum information processors|
+|[QUEKO](https://github.com/UCLA-VAST/QUEKO-benchmark)|UCLA|An open-source tool to generate benchmarks with the known optimal solutions|
+|[t\|ket>](https://cqcl.github.io/pytket/build/html/index.html#)|Cambridge Quantum|A proprietary and platform agnostic SDK for circuit optimisation, compilation and noise mitigation|
```

Comment:
Should we just call it tket or pytket?   I don't see the |ket> on the source website, and it makes it harder to read.

Reply:
I'm down - @karlunho strong opinions here?